paasta_tools/tron_tools.py

```diff
@@ -256,7 +257,15 @@ def get_schedule(self):
         return self.config_dict.get('schedule')
 
     def get_monitoring(self):
-        return self.config_dict.get('monitoring')
+        monitoring_conf = self.config_dict.get('monitoring', {})
```

Comment:
This isn't quite right.

I think this should use the existing `get_monitoring` function we have for instance_configs where we need it.
It isn't like we should use one or the other, it should "merge" the results of the two all the time. Setting page: true for job should be all you need (not have to also specify the team, which is already in monitoring.yaml)

I don't think this function of this object "should" read yet another yaml file though.

Reply:
I've modified `get_monitoring` to merge the tron monitoring config dict into the dict we get from `monitoring.yaml`.